jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_sorting_view.py

```diff
@@ -39,6 +40,7 @@ class SortingView(DOMWidget):
         sortingInfo = DictTrait(le.get_sorting_info(sorting_object=sorting.object(), recording_object=recording.object())).tag(sync=True)
         selection = DictTrait({}).tag(sync=True)
         curation = DictTrait({}).tag(sync=True)
+        externalUnitMetrics = ListTrait([]).tag(sync=True)
```

Comment:
externalUnitMetrics is now an attribute of the sorting view in jupyter. You should be able to set it either before or after the widget is created.